### DISCUSSION TITLE
Azure for Students account I don't know how to redeem my credit card.

### DISCUSSION THREAD

**Discussion by Tomohiro-yasue** [microsoft/SATechnicalOnboarding#143] [Questions]
(Alpha Student Ambassadors and above can redeem credits here: https://aka.ms/vsben) can I redeem my credit from on this site?

**Comment by AnthonyByansi** [comment]
Hi @Tomohiro-yasue 

Yes please. Ensure to redeem the credits through your MLSA account
![Visual-studio subscription](https://github.com/microsoft/SATechnicalOnboarding/assets/101401469/bddd7685-6148-41be-ad85-d2b0c5d39af6)

**Reply by AnthonyByansi** [reply]
Alternatively, you can sign-up for [Azure for students credits](https://azure.microsoft.com/free/students/?WT.mc_id=%3Fwt.mc_id%3Dstudentamb_260352) using you colleage/university email address
